util/stop/stopper_test.go

```diff
@@ -15,13 +15,19 @@
 //
 // Author: Spencer Kimball (spencer.kimball@gmail.com)
 
-package util
+package stop
 
 import (
 	"testing"
 	"time"
+
+	"github.com/cockroachdb/cockroach/testutils/fakeflags"
 )
 
+func init() {
+	fakeflags.InitLogFlags()
```

Comment:
I like having something to explicitly call. Don't like imports with side-effects, it's confusing enough as is. I'll change this package to use Ben's suggestion above.